i18n/chs/src/tree.i18n.json

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+// Do not edit this file. It is machine generated.
+{
+   "tree.build.target": "Build target: [{0}]",
```

Comment:
>[{0}] [](http://example.com/codeflow?start=39&length=5)

same suggestion here. I recommend you move the punctuation `[]` into the string being substituted.